File: sesion-5/secure-snippets-js-csharp/csharp/2-HardcodedSecrets.cs
// ✅ SEGURO: Usar variables de entorno
using System;

class SecretsSafe {
    static void Main() {
        var apiKey = Environment.GetEnvironmentVariable("API_KEY");
        var dbPass = Environment.GetEnvironmentVariable("DB_PASSWORD");
        Console.WriteLine(apiKey != null ? "API Key OK" : "API Key no definida");
    }
}

File: sesion-5/secure-snippets-js-csharp/js/2-hardcoded-secrets.js
/**
 * ✅ SEGURO: Usar variables de entorno
 */
require('dotenv').config();

const API_KEY = process.env.API_KEY;
const DB_PASSWORD = process.env.DB_PASSWORD;

function connect() {
  console.log("Conectando con API_KEY:", API_KEY ? "OK" : "No definida");
}
connect();
